flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/Catalog.java

```diff
@@ -505,6 +507,20 @@ void alterPartition(
     CatalogFunction getFunction(ObjectPath functionPath)
             throws FunctionNotExistException, CatalogException;
 
+    /**
+     * Get the procedure. Procedure name should be handled in a case insensitive way.
```

Comment:
Nit (suggested by Grammarly)
`Get the procedure. The procedure name should be handled in a case-insensitive way.`